python/cudf/cudf/dataframe/series.py

```diff
@@ -435,6 +435,24 @@ def values_to_string(self, nrows=None):
             out = ["" if v is None else str(v) for v in values]
         return out
 
+    def tolist(self):
+        """
+        Return a list type from series data.
+
+        Returns
+        -------
+        list
+
+        Difference from pandas:
+          * Returns numpy.datetime64 instead of pandas internal type
+        """
+        if isinstance(self.values, list):
+            return self.values
+        elif np.issubdtype(self.values.dtype, np.datetime64):
+            return list(self.values.astype(np.datetime64))
```

Comment:
@brandon-b-miller this should be `astype('datetime64[ns]')` if you want to cast to nanos. We can currently represent datetimes in s/ms/us/ns resolutions as `int64_t` unix timestamps, but default to `ms` when we don't know the resolution (as is the case with the np.datetime64 constructor).

Reply:
Ah nevermind, I missed that this is called on `self.values`. In that case, it probably makes sense to use `self.values.astype(self.dtype)`, since the DatetimeColumn's dtype already includes the resolution.

Or if you really do want to force it to be nanos first, you can do `list(self.astype('datetime64[ns]').values.astype(self.dtype))`

Reply:
updated - should just return a list of elements matching the datatype of `.values` now.

Reply:
Looks good, thanks!